webapp/components/interop-data.js

```diff
@@ -16,7 +16,15 @@ export const interopData = {
         'score_as_group': false
       }
     ],
-    'csv_url': 'https://raw.githubusercontent.com/web-platform-tests/results-analysis/gh-pages/data/interop-2021/interop-2021-{stable|experimental}-v2.csv',
+    /**
+     * Interop scores are "frozen" after the end of the year.
+     * Once an interop year is completed, results are generated one more time
+     * from the results-analysis script for the full year, and those scores
+     * are placed for reference in the webapp/static directory. The score
+     * is no longer updated and is referenced from this location.
+     * More information at https://github.com/web-platform-tests/results-analysis
+    **/
+    'csv_url': '/static/interop-2021-{stable|experimental}-v2.csv',
```

Comment:
Can we drop the v2 suffix now that the URLs are changing anyway? (I think we added them to make the switch between v1 and v2 smooth, and possibly as cache busting.)

Reply:
Yes, that's a good idea. It might also be worthwhile making this change eventually for the `results-analysis` repo.